modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -48,11 +48,17 @@ ComputeReturnMappingStress::initialSetup()
 }
 
 void
+ComputeReturnMappingStress::initQpStatefulProperties()
+{
+  ComputeFiniteStrainElasticStress::initQpStatefulProperties();
+  _elastic_strain_old[_qp] = _elastic_strain[_qp];
```

Comment:
This isn't correct (or maybe just not necessary since I'm not looking down into the BASE CLASS routine. You only need to set the current property in this routine. MOOSE copies it back for you.